schemas/twitter.js

```diff
@@ -0,0 +1,167 @@
+import * as twitter from '../apis/twitter';
+
+import {
+    GraphQLSchema,
+    GraphQLObjectType,
+    GraphQLID,
+    GraphQLString,
+    GraphQLNonNull,
+    GraphQLInt,
+    GraphQLList,
+} from 'graphql';
+
+let UserType = new GraphQLObjectType({
+
+    name: 'TwitterUser',
+    description: 'Twitter user, or as we like to say in France: a \'tweetos\'',
+    fields: () => ({
+        created_at: { type: GraphQLString },
```

Comment:
@clayallsopp, even with your example and after reading [ESLint doc about this spacing style](http://eslint.org/docs/rules/key-spacing.html) I'm not completely sure about the modifications I have to provide.
I'm about to update my PR with this kind of spacing: https://gist.github.com/mr-wildcard/bc743991439287b8c7bb
Is this correct or too much ? Please provide an example if it's not ok :)